src/TimeService.Business/Commands/Import/ImportStatCommand.cs

```diff
@@ -546,11 +550,11 @@ public OperationResultResponse<byte[]> Execute(ImportStatFilter filter)
 
       if (filter.DepartmentId.HasValue)
       {
-        usersIds = FindDepartmentUsers(filter.DepartmentId.Value, errors);
+        usersIds = FindDepartmentUsers(filter.DepartmentId.Value, new DateTime(filter.Year, filter.Month, 1), errors);
 
         projectsUsers = GetProjectsUsers(usersIds, errors);
 
-        projects = GetProjects(projectsUsers.Select(pu => pu.ProjectId).Distinct().ToList(), false, errors);
+        projects = GetProjects(projectsUsers?.Select(pu => pu.ProjectId).Distinct().ToList(), false, errors);
```

Comment:
if projectsUsers is null - will be crash

Reply:
created task 889